personal_project/ContentView.swift

```diff
@@ -1,66 +1,82 @@
-                    HStack{
-                        //image from json
-                        if let movies = Movie.movies {
-                            AsyncImage(url: URL(string: movies.Poster)) { image in
-                                    image
-                                    .resizable().aspectRatio(contentMode: .fit)
-                                    .scaledToFit().padding(.vertical).padding(.horizontal)
-                                   .frame(width: 100, height: 100, alignment: .bottomLeading)
+    @State var searchInput: String = ""
+    //change movie to an array of movies
+    @State var movie = Movie.movie
+    @State var movies = [Movie]()
 
-                            } placeholder: {
-                                Color.red
-                            } .frame(maxWidth: 80, alignment: .bottomLeading)
-                        }
-                        //categories from json
-                        if let movies = Movie.movies {
-                            let genreList = movies.Genre.split(separator: ",")
-                            Capsule().stroke(.white).frame(width: 70, height: 40).overlay(Text(genreList[0]).font(.system(size: 12.0)).foregroundColor(.white).padding(.vertical).padding(.horizontal))
-                            Capsule().stroke(.white).frame(width: 95, height: 40).overlay(Text(genreList[1]).font(.system(size: 12.0)).foregroundColor(.white).padding(.vertical).padding(.horizontal))
-                            Capsule().stroke(.white).frame(width: 85, height: 40).overlay(Text(genreList[2]).font(.system(size: 12.0)).foregroundColor(.white).padding(.vertical).padding(.horizontal))
-                        }
-                        
-                        
-                        
-                        
-                } .foregroundColor(Color("lightGrey"))
+    //change this to be a list of movies -> loop through movies and display each on a card
+    //list type in Swift UI that wraps the VStack
+    var body: some View {
+        
+        Text("Search").padding(20).font(.system(size: 28))
+        HStack{
+            Button {
+                //  print("\(search)")
+                API.fetchMovie(term: "\(searchInput)") { result in
+                    switch result {
+                    case .success(let movies):
+                        // print("hi")
+                        // print("got Title:", movies)
+                        self.movies = movies
+                        // print("got Year:", Year)
+                    case .failure(let error):
+                        print("Fetching Title failed.", error)
+                    }
+                }
+            } label: {
+                Image("searchIcon").resizable()
+                    .padding(15).frame(width: 45, height: 45)
+               // Text("search").padding(25)
             }
-            ZStack {
+            TextField("Search For A Movie", text: $searchInput)
```

Comment:
"Search for a Movie" would be the proper capitalization for a [title case](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) string here.

There are some converters like [this one](https://titlecaseconverter.com) that you can use, but the rules aren't too hard to understand and are worth knowing, IMO.